content/source/docs/cloud/sentinel/import/tfrun.html.md

```diff
@@ -9,13 +9,18 @@ description: |-
 
 The `tfrun` import provides access to data associated with a [Terraform run][run-glossary].
 
-This import currently includes two namespaces — `workspace`, which provides Sentinel policies with
-static data regarding the Terraform Cloud [workspace][workspace-glossary] associated with the run,
-and `cost_estimate`, which provides Sentinel policies with static data regarding the Terraform Cloud
-cost estimate associated with the run.
+This import currently consist of a `root` namespace, as well as namespaces for the `organization`, `workspace` and `cost-estimate`. Each namespace provides static data regarding the Terraform Cloud application that can then be consumed by Sentinel during a policy evaluation.
```

Comment:
Tying into https://github.com/hashicorp/terraform-website/pull/1063/files#r357488915.

I think something along the lines of `consists of run attributes, as well as...`.

Reply:
@dylanegan definitely a better approach considering the feedback thus far.

Reply:
https://github.com/hashicorp/terraform-website/pull/1063#discussion_r358310804

It looks like `root` is correct.